### PR TITLE
[react] Remove unstable_createMutableSource

### PR DESCRIPTION
**Pull request by eps1lon**
Was removed in https://github.com/facebook/react/pull/24209

### CODE REVIEW

File: types/vite-plugin-react-svg/index.d.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/visualfanatic/vite-svg
 // Definitions by: Priyanshu Rav <https://github.com/priyanshurav>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.8
+// Minimum TypeScript Version: 4.0
```
Comment (eps1lon) [inline types/vite-plugin-react-svg/index.d.ts:5]: Cherry-pick from https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59777

File: types/vite-plugin-react-svg/tsconfig.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6", "dom"
```
Comment (eps1lon) [inline types/vite-plugin-react-svg/tsconfig.json:5]: Cherry-pick from https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59777

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react@latest/))
was missing the following properties:
1. unstable_act
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1883e40ff826f08765d62aa3055f46d282902636
</p>

**Review by johnnyreilly — APPROVED** [review]

**Comment by eps1lon** [comment]
Ready to merge

**Pull request merged**